### PR TITLE
Add fish shell completions for monerod and monero-wallet-cli

### PR DESCRIPTION
**Pull request by LeoNero**
This PR adds fish shell completions for `monerod` and `monero-wallet-cli`. I am starting to use those binary files, and using `--help` all the time gets boring and slow. Maybe others that use Fish will find this useful too.

Basically, this is the output of `--help` adapted for Fish completions. I did a simple script for it and then manually imploved some things, such as adding `-a` flags, etc.

### CODE REVIEW

**Review by selsta — COMMENTED** [review]
I wonder if `utils/` makes more sense, the `systemd` example script is in there.

Also can you squash this to a single commit and name it something like: `utils: add fish shell completion`, that would be more in line with how commits are named in this project.

**Comment by LeoNero** [comment]
@selsta I'll do that when I get home. Thanks!

**Comment by LeoNero** [comment]
@selsta done :)

**Review by selsta — APPROVED** [review]
I use fish so that's useful for me, thanks :)

**Comment by LeoNero** [comment]
@selsta I just added completions for monero-wallet-rpc too.

**Comment by LeoNero** [comment]
lmao just added it. My bad

**Comment by LeoNero** [comment]
actually wait a second

File: utils/fish/monero-wallet-cli.fish [hunk 1]
```diff
@@ -0,0 +1,61 @@
+complete -c monero-wallet-cli -f
```
Comment (selsta) [inline utils/fish/monero-wallet-cli.fish:1]:
```suggestion
complete -c monero-wallet-cli -f
```

`monero-wallet-rpc` and `monerod` don't have this line, can you explain it?

Comment (LeoNero) [reply utils/fish/monero-wallet-cli.fish:1]: this means that file completions for the flags are disabled by default and, if a flag accepts a file path as an argument, we then need to use `-F` in its definition. I will put this line in the other files for consistency

Comment (LeoNero) [reply utils/fish/monero-wallet-cli.fish:1]: Actually, it is also important for the file completionst that find files with a specific suffix. SO it is necessary, will add it

**Comment by LeoNero** [comment]
Done :)

**Comment by selsta** [comment]
Screenshot 2022-07-01 at 01 04 08

Should this only suggest .json files? If yes, it does not seem to work yet.

**Comment by LeoNero** [comment]
Yea, I saw that. Per documentation of `__fish_complete_suffix`, it should just show the `.json` files first, and the other files after.

I think it can be improved, but I did not want to spend much time on that. I'll see if there is a fix to it.

**Review by selsta — APPROVED** [review]

**Pull request merged**